### PR TITLE
Prep for content refs

### PR DESCRIPTION
**Pull request by theengineear**
A couple more things I can factor out of my gargantuan PR 😁.

I'm getting flow errors locally on this... not sure why tbh!

### CODE REVIEW

File: packages/core/src/state/entities/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { makeKernelsRecord } from "./kernels";
 import { makeKernelspecsRecord } from "./kernelspecs";
 import { makeModalsRecord } from "./modals";
 
+export * from "./contents";
```
Comment (theengineear) [inline packages/core/src/state/entities/index.js:12]: I don't get it. I'm exporting all-the-things from `contents` and all-the-things are getting re-exported from `state/index.js` in this file... flow fails, but tests pass...

**Comment by theengineear** [comment]
^^ Not sure if you have light to shed on this flow error? Am I missing something dumb?

File: packages/core/src/state/entities/contents.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import * as Immutable from "immutable";
```
Comment (rgbkrk) [inline packages/core/src/state/entities/contents.js:2]: This file is missing an `@flow` at the top.

Comment (rgbkrk) [reply packages/core/src/state/entities/contents.js:2]: That was all that was missing here. Everything now passes!

**Comment by rgbkrk** [comment]
Now for me to actually review. 😄

**Comment by rgbkrk** [comment]
Looking good. This seems like a reasonable transition point.

File: packages/core/src/state/entities/contents.js (hunk 2 above)
Comment (theengineear) [reply packages/core/src/state/entities/contents.js:2]: face << palm. how many times...

**Comment by theengineear** [comment]
Apparently I was tired last night 😛

**Pull request merged**